src/pages/SignupHandler/index.js

```diff
@@ -119,13 +134,47 @@ const SignupHandler = props => {
     }
   };
 
-  const handleSignupSubmit = () => {
+  const handleSignupSubmit = async () => {
     const { dispatch } = props;
-    dispatch(routerRedux.push('/login'));
+    const response = await dispatch({
+      type: 'auth/registerUser',
+      payload: {
+        firstName,
+        lastName,
+        username: firstName.toLowerCase(),
+        password,
+        email,
+      },
+    });
+    const { message, status } = response;
+    setAlertComponent({
+      message,
+      variant: status === 'failure' ? 'danger' : 'success',
+    });
+    setAlertOpen(true);
+    if (status === 'success') {
+      // wait for a couple
+      // of seconds before taking the
+      // user to login page.
+      setTimeout(() => {
```

Comment:
In our design sync conversations, we mentioned populating the UI masthead with the username and removing the login banner prompt as enough visual feedback to let the user know they've successfully logged in. Can we consider removing this delay as a result?

Reply:
@gurbirkalsi That's right, the problem with this is that, if we do not add wait then the toast alert which is a part of this component won't show up (cause the user has already navigated to some other page). I am currently working on having the alert as a part of the parent component (most likely the AuthLayout component, will make a custom component if necessary which can be connected to the store for storing error messages as well). So, till that is done, let's keep it as it is. I will add it in the TODOs of this PR.